package/libskylark/src/libskylark.h

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Jacob McNamee <jacob@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#ifndef SWIFTNAV_LIBSKYLARK_H
+#define SWIFTNAV_LIBSKYLARK_H
+
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+// #include <sbp_settings.h>
+#include <sbp_zmq.h>
+
+/**
+ *  Logging utilities.
+ */
+
+#define log_info(...) fprintf(stdout, __VA_ARGS__)
```

Comment:
TODO: Do we have some other logging stuff somewhere we want to reuse? Also, what if we want to log out to SBP?